flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/partitioner/FlinkFixedPartitioner.java

```diff
@@ -50,6 +51,7 @@
  * To avoid such an unbalanced partitioning, use a round-robin kafka partitioner (note that this will
  * cause a lot of network connections between all the Flink instances and all the Kafka brokers).
  */
+@Internal
```

Comment:
Not sure here. This could be `@PublicEvolving`.

It could very well be that the user simply instantiates a `FlinkFixedPartitioner` as the provided custom partitioner.